ghga_transpiler/config/configs/1.0.0.yaml

```
@@ -0,0 +1,70 @@
+ghga_metadata_version: 1.0
```

```suggestion
ghga_metadata_version: 0.0.1
```
Shall we call this one 0.0.1? Like this we can play around and be sure we'll never clash with a real future version since we've already skipped and never used this version. Would need to be renamed, too.